src/autocomplete.js

```diff
@@ -55,10 +55,13 @@ class Autocomplete {
     constructor() {
         this.autoInsert = false;
         this.autoSelect = true;
+        this.autoShown = false;
         this.exactMatch = false;
         this.inlineEnabled = false;
         this.keyboardHandler = new HashHandler();
         this.keyboardHandler.bindKeys(this.commands);
+        this.emptyMessage = null;
```

Comment:
It may be better to not set this to null, to allow  setting the function on prototype